blueocean-git-pipeline/pom.xml

```diff
@@ -24,8 +24,13 @@
       <artifactId>blueocean-pipeline-scm-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>blueocean-pipeline-api-impl</artifactId>
+    </dependency>
+    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
+      <version>3.5.1-SNAPSHOT</version>
```

Comment:
Better to `mvn -DskipTests clean deploy` a snapshot and use the timestamped version here. Otherwise your CI build is at the mercy of anyone else who might have been messing around with unrelated patches to the upstream plugin.